view/theme/frio/css/style.css

```diff
@@ -2515,3 +2515,23 @@ body .tread-wrapper .hovercard:hover .hover-card-content a {
 section .profile-match-wrapper {
     float: left;
 }
+
+/* Pagination improvements */
+
+.pagination > li > a,
+.pagination > li > span {
+	color: $link_color;
+}
+.pagination>.active>a,
+.pagination>.active>a:focus,
+.pagination>.active>a:hover,
+.pagination>.active>span,
+.pagination>.active>span:focus,
+.pagination>.active>span:hover {
+	background-color: $link_color;
+    border-color: $link_color;
+}
+
+.disabled > a {
+	pointer-events: none;
+}
```

Comment:
can you please try to put this css not at the end of the file. I don't know which place would be fit best (maybe some lines above) but at the end of frios css file should be hacks or css which can be deleted sometime in the future